ddl/ddl.go

```diff
@@ -374,6 +379,10 @@ func (d *ddl) genGlobalID() (int64, error) {
 	return globalID, errors.Trace(err)
 }
 
+func (d *ddl) SchemaVersionSyncer() *schemaVersionSyncer {
```

Comment:
This structure isn't exported, so this can not comment. And other interfaces have no comments too.